src/js/components/Drop/DropContainer.js

```diff
@@ -37,6 +37,7 @@ const DropContainer = forwardRef(
   (
     {
       align = defaultAlign,
+      setAlignControlMirror,
```

Comment:
Can we call this `onAlign`? I think that will be more consistent and generic, caller is notified when the actual alignment changes.